app/views/state_file/questions/id_grocery_credit_review/edit.html.erb

```diff
@@ -0,0 +1,40 @@
+<% title = t('.title', grocery_credit_amount: grocery_credit_amount) %>
+
+<% content_for :page_title, t('.title', grocery_credit_amount: grocery_credit_amount) %>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= t('.title_html', grocery_credit_amount: grocery_credit_amount) %></h1>
+
+  <%= form_with model: @form, url: { action: :update }, method: :put, local: true, builder: VitaMinFormBuilder, class: 'form-card' do |f| %>
+    <div class="white-group">
+      <p class="spacing-below-0"><%= t(".would_you_like_to_donate") %></p>
+      <%=
+        f.cfa_radio_set(
+          :donate_grocery_credit,
+          collection: [
+            { value: :yes, label: t("general.affirmative") },
+            { value: :no, label: t("general.negative") },
+          ]
+        )
+      %>
+
+      <div class="notice spacing-below-0">
+        <p><%= t('.donate_warning_p1') %>
+        <p><%= t('.donate_warning_p2') %>
+      </div>
+    </div>
+
+    <div class="reveal">
+      <p><a href="#" class="reveal__link"><%= t('.donate_impact_heading') %></a></p>
```

Comment:
change to `<button class="reveal__button">` after merging in main

Reply:
@mrotondo bump

Reply:
done